docker/database-setup/17-create-logbooks-tags.sql

```diff
@@ -0,0 +1,7 @@
+create table if not exists logbooks_tags (
```

Comment:
This isn't used anywhere yet. There will probably be more tables like this, for logs and maybe even for item templates and items, but that's follow-up work.